### PR TITLE
Create menu

### PR DESCRIPTION
**Pull request by cinekqw1**

### CODE REVIEW

File: src/menu.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
+
+var title;
+
+class MainMenu extends Phaser.Scene{
+
+
```
Comment (fturek) [inline src/menu.js]: We can remove unnecessary new lines here.

File: src/menu.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+      }, this)
+     .on('pointerover', function (event) {
+       this.clickButton.setStyle({ fill: '#ff0'});
+     }, this)
+     .on('pointerout', function (event) {
+       this.clickButton.setStyle({ fill: '#0f0' });
+     },this );
+
+  }
+
```
Comment (fturek) [inline src/menu.js]: We can remove unnecessary new lines here.

**Review by fturek — APPROVED** [review]
LGTM!

**Comment by pparuzel** [comment]
Resolved #10 
Resolved #11

**Pull request merged**